Mathlib/RingTheory/Noetherian.lean

```diff
@@ -571,6 +571,11 @@ theorem isNoetherian_of_fg_of_noetherian' {R M} [Ring R] [AddCommGroup M] [Modul
   isNoetherian_of_linearEquiv (LinearEquiv.ofTop (⊤ : Submodule R M) rfl)
 #align is_noetherian_of_fg_of_noetherian' isNoetherian_of_fg_of_noetherian'
```

Comment:
I think `isNoetherian_of_fg_of_noetherian'` should just be removed, it's just a weird way of writing the instance. Can you inline the proof into the instance?